Console error (CraftItemEvent)

> [19:11:11] [Server thread/ERROR]: Could not pass event CraftItemEvent to DuelMe v0.17.4
> org.bukkit.event.EventException
>         at org.bukkit.plugin.java.JavaPluginLoader$1.execute(JavaPluginLoader.java:306) ~[spigot-1.11.2-R0.1-SNAPSHOT.jar:git-Spigot-7d78b81-775f1e1]
>         at org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:62) ~[spigot-1.11.2-R0.1-SNAPSHOT.jar:git-Spigot-7d78b81-775f1e1]
>         at org.bukkit.plugin.SimplePluginManager.fireEvent(SimplePluginManager.java:502) [spigot-1.11.2-R0.1-SNAPSHOT.jar:git-Spigot-7d78b81-775f1e1]
>         at org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:487) [spigot-1.11.2-R0.1-SNAPSHOT.jar:git-Spigot-7d78b81-775f1e1]
>         at net.minecraft.server.v1_11_R1.PlayerConnection.a(PlayerConnection.java:1839) [spigot-1.11.2-R0.1-SNAPSHOT.jar:git-Spigot-7d78b81-775f1e1]
>         at net.minecraft.server.v1_11_R1.PacketPlayInWindowClick.a(SourceFile:33) [spigot-1.11.2-R0.1-SNAPSHOT.jar:git-Spigot-7d78b81-775f1e1]
>         at net.minecraft.server.v1_11_R1.PacketPlayInWindowClick.a(SourceFile:10) [spigot-1.11.2-R0.1-SNAPSHOT.jar:git-Spigot-7d78b81-775f1e1]
>         at net.minecraft.server.v1_11_R1.PlayerConnectionUtils$1.run(SourceFile:13) [spigot-1.11.2-R0.1-SNAPSHOT.jar:git-Spigot-7d78b81-775f1e1]
>         at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) [?:1.8.0_111]
>         at java.util.concurrent.FutureTask.run(FutureTask.java:266) [?:1.8.0_111]
>         at net.minecraft.server.v1_11_R1.SystemUtils.a(SourceFile:46) [spigot-1.11.2-R0.1-SNAPSHOT.jar:git-Spigot-7d78b81-775f1e1]
>         at net.minecraft.server.v1_11_R1.MinecraftServer.D(MinecraftServer.java:739) [spigot-1.11.2-R0.1-SNAPSHOT.jar:git-Spigot-7d78b81-775f1e1]
>         at net.minecraft.server.v1_11_R1.DedicatedServer.D(DedicatedServer.java:399) [spigot-1.11.2-R0.1-SNAPSHOT.jar:git-Spigot-7d78b81-775f1e1]
>         at net.minecraft.server.v1_11_R1.MinecraftServer.C(MinecraftServer.java:675) [spigot-1.11.2-R0.1-SNAPSHOT.jar:git-Spigot-7d78b81-775f1e1]
>         at net.minecraft.server.v1_11_R1.MinecraftServer.run(MinecraftServer.java:574) [spigot-1.11.2-R0.1-SNAPSHOT.jar:git-Spigot-7d78b81-775f1e1]
>         at java.lang.Thread.run(Thread.java:745) [?:1.8.0_111]
> Caused by: java.lang.NullPointerException				
>         at com.github.teozfrank.duelme.menus.AcceptMenu.onInventoryClick(AcceptMenu.java:75) ~[?:?]
>         at sun.reflect.GeneratedMethodAccessor448.invoke(Unknown Source) ~[?:?]
>         at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:1.8.0_111]
>         at java.lang.reflect.Method.invoke(Method.java:498) ~[?:1.8.0_111]
>         at org.bukkit.plugin.java.JavaPluginLoader$1.execute(JavaPluginLoader.java:302) ~[spigot-1.11.2-R0.1-SNAPSHOT.jar:git-Spigot-7d78b81-775f1e1]
>         ... 15 more

--- a/Plugin/src/main/java/com/github/teozfrank/duelme/menus/AcceptMenu.java
+++ b/Plugin/src/main/java/com/github/teozfrank/duelme/menus/AcceptMenu.java
@@ -72,7 +72,7 @@ public class AcceptMenu implements Listener {
 
     @EventHandler(priority = EventPriority.NORMAL)
     public void onInventoryClick(InventoryClickEvent e){
-        if(!e.getInventory().getName().equalsIgnoreCase(acceptMenu.getName())){
+        if(acceptMenu == null || !e.getInventory().getName().equalsIgnoreCase(acceptMenu.getName())){
             return;
         }
 

